web/src/user_guide/Page.svelte

```diff
@@ -0,0 +1,518 @@
+<script lang="ts">
+  import "@picocss/pico/css/pico.conditional.jade.min.css";
+  import Toc from "svelte-toc";
+  import About from "../About.svelte";
+  import image01 from "./image01.webp?url";
+  import image02 from "./image02.webp?url";
+  import image03 from "./image03.webp?url";
+  import image04 from "./image04.webp?url";
+  import image05 from "./image05.webp?url";
+  import image06 from "./image06.webp?url";
+  import image07 from "./image07.webp?url";
+  import image08 from "./image08.webp?url";
+  import image09 from "./image09.webp?url";
```

Comment:
The original images were pngs, up to 2MB each. Webp is tiny and looks identical, and https://caniuse.com/webp says it's stable since 2020.

Reply:
If you ever do need to use png, I've seen some moderate improvements by using something like pngquant. 

Though I assume webp compresses even better.